Source/FlowEditor/Public/Graph/FlowGraphEditorSettings.h

```diff
@@ -13,6 +14,24 @@ enum class EFlowNodeDoubleClickTarget : uint8
 	PrimaryAssetOrNodeDefinition UMETA(Tooltip = "First try opening the asset then if there is none, open the node class") 
 };
 
+USTRUCT()
+struct FLOWEDITOR_API FFlowTraitSettings
+{
+	GENERATED_BODY()
+	
+	UPROPERTY()
+	TArray<FFlowDebugTrait> NodeTraits;
+
+	UPROPERTY()
+	TArray<FFlowDebugTrait> PinTraits;
```

Comment:
Store as TArray and not as TMap<FGuid, FFlowDebugTrait>, because usually there will be very small amount of traits, and I think TArray would be better choice as all its elements are small and grouped in the same memory, so it is more cache-friendly.